app/services/reject_by_default_feedback.rb

```diff
@@ -32,4 +34,24 @@ def notify_slack
 
     SlackNotificationWorker.perform_async(message, url)
   end
+
+  def not_applied_again?
+    ApplicationForm
+      .where(candidate_id: candidate.id, recruitment_cycle_year: RecruitmentCycle.current_year, phase: 'apply_2')
+      .empty?
+  end
+
+  def unsuccessful_application_choices?
+    ApplicationForm
+      .includes(:application_choices)
+      .where(candidate_id: candidate.id, recruitment_cycle_year: RecruitmentCycle.current_year)
+      .first
```

Comment:
Rather than selecting the first application, should we select the one which this application choice belongs to? I think we'd want to display the same guidance if the candidate gets rejected by default on an apply again application?

Reply:
Yep good spot. I've pushed the change 👍